src/js/utils/events.js

```diff
@@ -273,7 +290,13 @@ export function on(elem, type, fn) {
 
   if (data.handlers[type].length === 1) {
     if (elem.addEventListener) {
-      elem.addEventListener(type, data.dispatcher, false);
+      let options = false;
+
+      if (_supportsPassive &&
+        ['touchstart', 'touchmove'].indexOf(type) > -1) {
```

Comment:
I wonder if this array ought to be defined once in the parent scope rather than inline. Something like:

```js
const passiveEvents = [
  'touchstart',
  'touchmove'
];
```